variants/esp32s3/heltec_v4/platformio.ini

```diff
@@ -7,3 +7,107 @@ build_flags =
   ${esp32s3_base.build_flags}
   -D HELTEC_V4
   -I variants/esp32s3/heltec_v4
+lib_deps = 
+  ${esp32s3_base.lib_deps}
+
+
+[env:heltec-v4-oled]
```

Comment:
This should be heltec-v4, as it will break the existing web-flasher and install scripts.